sdk/src/lib.rs

```diff
@@ -20,6 +20,14 @@
 //! This library supports reading, creating and embedding C2PA data
 //! with JPEG and PNG images.
 //!
+//! To read or write a manifest file, you must add the `file_io` dependency to your Cargo.toml.
+//! EXCEPTION: If you are building for WASM, do not add this dependency. 
+//! For example:
+//!
+//! ```
```

Comment:
This is probably what is failing because it's being parsed as a Rust code example that should be tested. I'll need to get back to you on the proper syntax for excluding this from being executed in this way.

Reply:
Since line 28 isn't Rust code, you'll need to tell the doc test engine to not attempt to run it as such. Per https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes, you can change line 27 to (three backticks)`text` and that should pass.

Reply:
Did that. S/be fixed I hope.

Reply:
@scouten-adobe Hm, it's still failing...  If nothing else, I could put it inline, e.g. `c2pa = {version="0.11.0", features=["file_io"]}` unless there is a better way.